hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -686,19 +756,19 @@ Sizes below are specified in multiples of 32bit words.
-|       |                                 |             |                | {{< regref "SECRET0_DIGEST_0" >}}        | 0x728        | 8
-| 5     | SECRET1                         | 88          | 64bit          | FLASH_ADDR_KEY                           | 0x730        | 32
-|       |                                 |             |                | FLASH_DATA_KEY                           | 0x750        | 32
-|       |                                 |             |                | SRAM_DATA_KEY                            | 0x770        | 16
-|       |                                 |             |                | {{< regref "SECRET1_DIGEST_0" >}}        | 0x780        | 8
-| 6     | SECRET2                         | 120         | 64bit          | RMA_TOKEN                                | 0x788        | 16
-|       |                                 |             |                | CREATOR_ROOT_KEY_SHARE0                  | 0x798        | 32
-|       |                                 |             |                | CREATOR_ROOT_KEY_SHARE1                  | 0x7B8        | 32
-|       |                                 |             |                | {{< regref "SECRET2_DIGEST_0" >}}        | 0x7F8        | 8
+| 3     | SECRET0                         | 40          | 64bit          | TEST_UNLOCK_TOKEN                        | 0x6B0        | 16
+|       |                                 |             |                | TEST_EXIT_TOKEN                          | 0x6C0        | 16
+|       |                                 |             |                | {{< regref "SECRET0_DIGEST_0" >}}        | 0x6D0        | 8
+| 4     | SECRET1                         | 88          | 64bit          | FLASH_ADDR_KEY                           | 0x6D8        | 32
+|       |                                 |             |                | FLASH_DATA_KEY                           | 0x6F8        | 32
+|       |                                 |             |                | SRAM_DATA_KEY                            | 0x718        | 16
+|       |                                 |             |                | {{< regref "SECRET1_DIGEST_0" >}}        | 0x728        | 8
+| 5     | SECRET2                         | 120         | 64bit          | RMA_TOKEN                                | 0x730        | 16
+|       |                                 |             |                | CREATOR_ROOT_KEY_SHARE0                  | 0x740        | 32
+|       |                                 |             |                | CREATOR_ROOT_KEY_SHARE1                  | 0x760        | 32
+|       |                                 |             |                | {{< regref "SECRET2_DIGEST_0" >}}        | 0x7A0        | 8
+| 6     | LIFE_CYCLE                      | 88          | 32bit          | {{< regref "LC_STATE_0" >}}              | 0x7A8        | 24
```

Comment:
@mcy FYI

Reply:
The ordering of the partitions isn't a big deal to us on the interface side; the implementation will handle all of this transparently.

Relatedly, are there constants for the starts and ends of the various partitions? Those would be nice to have if we don't already.

Reply:
Ok, good to know, thanks.

RE the constants: Not yet, but I am going to put this into an hjson such that the register file, documentation and SV package can be generated from one common source. As part of that, I can propagate the start/stop offsets into the `otp_ctrl.hjson` such that they become available as constants in the generated header. Would that work?

Reply:
Hey Michael, I think since the order of the partition changed, you may also want to update the info in register file [here](https://github.com/lowRISC/opentitan/blob/master/hw/ip/otp_ctrl/data/otp_ctrl.hjson#L509).

Reply:
Thanks for pointing this out @cindychip!

I believe this is taken care of already (in this PR). The documentation of that CSR is going to change to:

```
For the digest calculation command, set this register to the partition base offset.
```

I.e., instead of writing indices, you just set the address CSR to the partition base offset.